### DISCUSSION TITLE
[Feature] declare AnimatedIcon.SetState(....) in Xaml directly

### DISCUSSION THREAD

**Discussion by AbdAlghaniAlbiek** [CommunityToolkit/Labs-Windows#250] [Ideas]
## Describe the problem this feature would solve
Is there any way to set states of an AnimatedIcon control directly in xaml without making pointer events and do all of this in code behind ????

## Describe the solution
¯\_(ツ)_/¯

## Describe alternatives you've considered
¯\_(ツ)_/¯

## Additional context & Screenshots
![Screenshot (44)_LI](https://user-images.githubusercontent.com/37735487/143782032-4d144fe6-ceb5-49f0-8f40-0d6c7a686ceb.jpg)

**Comment by ghost** [comment]
Hello, 'AbdAlghaniAlbiek! Thanks for submitting a new feature request. I've automatically added a vote 👍 reaction to help get things started. Other community members can vote to help us prioritize this feature in the future!

**Comment by michael-hawker** [comment]
@AbdAlghaniAlbiek seems like a Behavior could be the solution here to simplify within XAML easily enough.

Only thing right now is our Behaviors package doesn't include a dependency on the WinUI library... 🤔

**Reply by Arlodotexe** [reply]
Definitely needs to be a behavior. We'll have a better chance of experimenting with this now that we're in Labs!

**Comment by michael-hawker** [comment]
Looks like @vgromfeld is also needing the reference for #4399, so that may also push us to an 8.0 update (along with the .NET Community Toolkit split) and this would make a great inclusion for that as well. Will need to figure out a plan forward.
